### ISSUE TITLE
Entity workflow buttons isn't rendered if __toString of entity return empty string

### ISSUE THREAD

**Issue by scapital** [oroinc/platform#817]
Problem in [this line](https://github.com/oroinc/platform/blob/master/src/Oro/Bundle/WorkflowBundle/Resources/views/Widget/entityWorkflows.html.twig#L1)

As [docs](https://twig.symfony.com/doc/2.x/tests/empty.html) say,
> For objects that implement the __toString() magic method (and not Countable), it will check if an empty string is returned

So, if __toString of entity return empty string, we won't see workflow buttons.

Maybe, refactor this template to something like that:
```
{% set entity_class = oro_class_name(entity) %}
{% if entity_class is not null and has_workflows(entity) %}
```

As [TwigExtension](https://github.com/oroinc/platform/blob/master/src/Oro/Bundle/EntityBundle/Twig/EntityExtension.php#L102) has a checking "is_object", it should work fine in any cases.

Thanks

**Comment by mbessolov** [comment]
Thank you for the suggestion. We'll check if it can be done this way and will include it in one of the next releases.

(internal reference - BAP-16680 )

**Comment by anyt** [comment]
Fixed in this commit https://github.com/oroinc/platform/commit/061e79bb74d15153f9b98d5efb9cab289fbb0d11
And already released in `3.0.0-beta`.

Thank you @scapital.
